packages/trip-details/src/index.tsx

```diff
@@ -256,150 +47,22 @@ export function TripDetails({
         />
       </S.TripDetailsHeader>
       <S.TripDetailsBody>
-        <TripDetail
-          // Any custom description for the Departure message needs to be handled by the slot.
-          description={
-            DepartureDetails && (
-              <DepartureDetails departureDate={departureDate} />
-            )
-          }
-          icon={<CalendarAlt size={17} />}
-          summary={
-            <S.Timing>
-              <FormattedMessage
-                defaultMessage={defaultMessages["otpUi.TripDetails.departure"]}
-                description="Text showing the departure date/time for a trip."
-                id="otpUi.TripDetails.departure"
-                values={{
-                  departureDate,
-                  strong: boldText
-                }}
-              />
-            </S.Timing>
-          }
+        <DepartureDate
+          itinerary={itinerary}
+          DepartureDetails={DepartureDetails}
+        />
+        <Fares
+          itinerary={itinerary}
+          fareConfig={fareConfig}
+          FareDetails={FareDetails}
+        />
+        <TimeActive
```

Comment:
The little question mark isn't working in some circumstances

Reply:
Going to need more info on what circumstances you're talking about. It works for me 
<img width="556" alt="image" src="https://github.com/user-attachments/assets/d845e1f5-3797-4ab1-ae1c-b56233e3f7e2" />

Reply:
<img width="539" alt="image" src="https://github.com/user-attachments/assets/8ea9f002-2ac1-4cfd-9cbc-9e0a73b8d9c6" />

Reply:
Did you try using the updated component inside otp-rr?

Reply:
It works fine in the story but not inside the app

Reply:
<img width="547" alt="image" src="https://github.com/user-attachments/assets/4b399ac1-6a15-4237-a404-1636f4381297" />
shown inside otp-rr...